lib/matplotlib/rcsetup.py

```diff
@@ -530,20 +530,28 @@ def validate_ps_distiller(s):
                                         'top', 'none'])
 validate_fillstylelist = _listify_validator(validate_fillstyle)
 
-validate_negative_linestyle = ValidateInStrings('negative_linestyle',
-                                                ['solid', 'dashed'],
-                                                ignorecase=True)
+_validate_negative_linestyle = ValidateInStrings('negative_linestyle',
+                                                 ['solid', 'dashed'],
+                                                 ignorecase=True)
 
 
+@deprecated('3.0.0?', pending=True,
```

Comment:
Yes :D 
I suggest deprecating this in 2.1, which means no backporting. Is that a problem?